networking/hardware_networks/configuring-hardware-offloading.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="configuring-hardware-offloading"]
+= Configuring hardware offloading
+include::modules/common-attributes.adoc[]
+:context: configuring-hardware-offloading
+
+toc::[]
+
+As a cluster administrator, you can configure hardware offloading on compatible nodes to increase data processing performance and reduce load on host CPUs.
+
+//What is this feature, who needs it?
+
+include::modules/nw-sriov-hwol-about-hardware-offloading.adoc[leveloffset=+1]
+
+//Which NICs do we support?
+
+include::modules/nw-sriov-hwol-supported-devices.adoc[leveloffset=+1]
+
+[id="configuring-hardware-offloading-prerequisites"]
+== Prerequisites
+
+* Your cluster has at least one bare metal machine with a network interface controller that is supported for hardware offloading.
+* You xref:/installing-sriov-operator.adoc#installing-sr-iov-operator_installing-sriov-operator[installed the SR-IOV Network Operator].
```

Comment:
This link is broken in the published output.

Reply:
Fixed.